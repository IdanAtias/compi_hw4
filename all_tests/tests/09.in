bool b;
while(b)
	break;

