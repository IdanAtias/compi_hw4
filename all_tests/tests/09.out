-- end scope --
var:b:0:BOOL
